app/assets/stylesheets/components/_card.scss

```diff
@@ -30,3 +30,19 @@
 .card-product .card-product-infos {
   padding: 16px;
 }
+
+.card-img-top {
```

Comment:
I think editing these styles could be an issue because you seem to be modifying bootstrap classes that are used on other pages. It might be better to work around this by simply adding bootstrap utility classes to the elements you want to change.

Reply:
I checked that .card-img-top, .card-title and .card-body where not used anywhere before I added them in the _card.css file.

Reply:
They are used on a different branch, that's not merged into master yet. I don't think overwriting bootstrap classes is a good idea. I think it's better to create a new class and append it to the card. Bootstrap also offers some utility classes that take care of things like `flex`, `padding`, `margin` etc

Reply:
To clarify, I think modifying bootstrap classes is a bad idea because down the line they might be implemented on different sections of the page (like on the `style` branch). Overwriting those can cause unexpected issues that will cause problems and will difficult to solve.